params.yml

```diff
@@ -187,6 +187,16 @@ args:
   # The fork id of the new rollup. It indicates the prover (zkROM/executor) version.
   zkevm_rollup_fork_id: 9
 
+  # The address of the rollup manager contract on L1.
+  # If specified, the rollup data will be retrieved from this contract instead of using the contracts service.
+  # zkevm_rollup_manager_address: ""
+  # # The block number at which the rollup manager contract was deployed.
+  # zkevm_rollup_manager_block_number: 0
+  # # The address of the global exit root contract on L2.
+  # zkevm_global_exit_root_l2_address: ""
+  # # The address of the Polygon data committee contract on L1.
+  # polygon_data_committee_address: ""
```

Comment:
This can be retrieved from the chain I guess so I still need to improve it but that's a good starting point